R/guide-colorbar.r

```diff
@@ -273,6 +295,16 @@ guide_gengrob.colorbar <- function(guide, theme) {
                rectGrob(x = 0, y = by, vjust = 0, hjust = 0, width = barwidth.c, height = bh, default.units = "mm",
                         gp = gpar(col = NA, fill = guide$bar$colour))
              })
+    }
+
+  # make frame around color bar if requested (colour is not NULL)
+  if (!is.null(guide$frame.colour)) {
+    grob.bar <- grobTree(grob.bar,
```

Comment:
Would you mind adhering to modern tidyverse style here?

```R

grobTree(
  grob.bar,
  rectGrob(
    width = barwidth.c, height = barheight.c, default.units = "mm",
    gp = gpar(
      col = guide$frame.colour,
      lwd = guide$frame.linewidth,
      lty = guide$frame.linetype,
      fill = NA
    )
  )
)
```

or similar

Reply:
Sure, will do.